### PR TITLE
Switch to go1.11 

### PR DESCRIPTION
**Pull request by soltysh**
Followup to https://github.com/openshift/origin/pull/22523
/assign @smarterclayton

### CODE REVIEW

**Comment by bparees** [comment]
@soltysh seems worthy of an aos-devel announcement.

**Comment by soltysh** [comment]
@bparees there will be one when we'll be close, it's a few days away still, just starting :wink:

File: origin.spec [hunk 1]
```diff
@@ -6,7 +6,7 @@
 %global gopath      %{_datadir}/gocode
 %global import_path github.com/openshift/origin
 
-%global golang_version 1.10
+%global golang_version 1.11
```
Comment (smarterclayton) [inline origin.spec]: Check with @sosiouxme on this change

Comment (smarterclayton) [reply origin.spec]: You probably will have to open a pr to ocp-build-data

Comment (soltysh) [reply origin.spec]: Will do.

Comment (soltysh) [reply origin.spec]: It's already there.

**Comment by sosiouxme** [comment]
*should* be fine in OCP. go-toolset-1.11 and shim are available.

File: origin.spec (hunk 1 above)
Comment (sosiouxme) [reply origin.spec]: apparently i forgot to comment here.
the spec file requesting 1.11 should suffice to get it.
image configs in ocp-build-data should already be updated for when this lands.

**Comment by sosiouxme** [comment]
This seems a bit more persistent than your average flake...

**Comment by soltysh** [comment]
> This seems a bit more persistent than your average flake...

So it looks, lemme check, until then
/hold

**Comment by soltysh** [comment]
Hmmm... none of the failures seems related to this PR in particular. Most of them are related to unauthorized errors, none of which are coming from these changes. 
/hold cancel
/retest

**Comment by soltysh** [comment]
Closing in favor of #22666.

**Pull request closed without merging**